src/main/java/de/cyface/collector/MainVerticle.java

```diff
@@ -107,10 +112,14 @@ private Router setupRoutes() {
 	}
 
 	private void startHttpServer(final Router router, final Future<Void> startFuture) {
-		String certificateFile = this.getClass().getResource("/localhost.jks").getFile();
-		HttpServerOptions options = new HttpServerOptions().setSsl(true).setKeyStoreOptions(new JksOptions().setPath(certificateFile).setPassword("secret"));
-		
-		vertx.createHttpServer(options).requestHandler(router::accept).listen(config().getInteger("http.port", 8080),
+		final int httpPort = Parameter.HTTP_PORT.intValue(vertx, 8080);
+		final String defaultCertificateFile = this.getClass().getResource("/localhost.jks").getFile();
```

Comment:
that's just the public key, right?
I'm wondering why here is a hard coded string when the parameters can be provided (see Parameter.java)